server/chat-plugins/hangman.ts

```diff
@@ -3,6 +3,15 @@
 * By bumbadadabum and Zarel. Art by crobat.
 */
 import {Utils} from '../../lib/utils';
+import {FS} from '../../lib/fs';
+
+const HANGMAN_FILE = 'config/chat-plugins/hangman.json';
+
+export let hangmanData: {[roomid: string]: {[question: string]: string[]}} = {};
```

Comment:
```suggestion
export let hangmanData: {[roomid: string]: {[phrase: string]: string[]}} = {};
```

this should be formatted as
```javascripts
roomid: {
	"this is a phrase": ["this is a possible hint", "this is another hint for the same phrase"],
	"this is another phrase": ["this is a possible hint", "this is another hint for the same phrase"],
},
```